api/yorkie/v1/admin.proto

```diff
@@ -28,6 +28,8 @@ option java_package = "dev.yorkie.api.v1";
 service AdminService {
   rpc SignUp(SignUpRequest) returns (SignUpResponse) {}
   rpc LogIn(LogInRequest) returns (LogInResponse) {}
+  rpc DeleteAccount(DeleteAccountRequest) returns (DeleteAccountResponse) {}
```

Comment:
I'm thinking of changing `SignUp` to `CreateAccount` or other ways to keep consistency of RPC names. How do you think about this?

Reply:
I think it would be better to use User instead of Account as @krapie suggested.

So I'm going to use `CreateUser` instead of `SignUp`. thank you for comment! 👍👍